bigquery/indexer.py

```diff
@@ -0,0 +1,114 @@
+"""Loads BigQuery table into Elasticsearch."""
+
+import csv
+import json
+import logging
+import os
+import time
+
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import bulk
+import pandas as pd
+
+
+# Log to stderr.
+logging.basicConfig(level=logging.INFO,
+    format='%(asctime)s %(filename)10s:%(lineno)s %(levelname)s %(message)s',
+    datefmt='%Y%m%d%H:%M:%S')
+logger = logging.getLogger('indexer.bigquery')
+
+
+def open_and_return_json(file_path):
+  """Opens and returns JSON contents.
+
+  Args:
+    file_path: Relative path of JSON file.
+
+  Returns:
+    Parsed JSON.
+  """
+  with open(file_path,'r') as f:
+    # Remove JSON comments.
+    jsonDict = json.loads('\n'.join([row for row in f.readlines() if len(row.split('//')) == 1]))
+  return jsonDict
+
+
+def init_elasticsearch(index_name):
+  es = Elasticsearch(['http://localhost:9200'], timeout=600)
```

Comment:
Since we'll probably need to swap out the hostname by environment, maybe provide this as a parameter?

Reply:
Done